utils.go

```diff
@@ -82,17 +83,39 @@ func GetRootlessRuntimeDir(rootlessUID int) (string, error) {
 	return path, nil
 }
 
+func getInitCommand() (string, error) {
+	file, err := os.Open("/proc/1/comm")
+	if err != nil {
+		return "", err
+	}
+	defer file.Close()
+	buf, err := ioutil.ReadAll(file)
+	if err != nil {
+		return "", err
+	}
```

Comment:
could you use directly `ioutil.ReadFile(...)` here?

Reply:
done